README.md

```diff
@@ -25,63 +25,64 @@ The I2Cwrapper framework consists of **four basic components.** The first two dr
    - a controller **interrupt** mechanism, 
    - triggering a **target reset**.
 2. **Firmware modules** that implement the actual functionality of the target device, e.g. controlling stepper and/or servo motors.
-   - Modules exist as separate **include files** (e.g. `ServoI2C_firmware.h`) and are **selected** for compilation via the `firmware_modules.h` file.
+   - Modules exist as separate **include files** (e.g. [`ServoI2C_firmware.h`](firmware/ServoI2C_firmware.h) ) and are **selected** for compilation via the [`firmware_modules.h`](firmware/firmware_modules.h) file.
    - Modules don't have to worry about the I2C overhead but can **concentrate on what's important**: interpreting and reacting to the controller device's commands and requests.
    - Modules can "inject" their code at different places in the firmware, so that there is a **high degree of flexibility**.
 
 The other two basic components are for the **I2C controller's side**:
-3. The **I2Cwrapper class**, provided by the `I2Cwrapper.h` library. 
+
+1. The [**I2Cwrapper class**](https://ftjuh.github.io/I2Cwrapper/class_i2_cwrapper.html), provided by the `I2Cwrapper.h` library. 
```

Comment:
Move "library" outside code markup, as per email.

Reply:
I think it's still inside code markup.